apps/prairielearn/src/components/QuestionScore.tsx

```diff
@@ -246,12 +246,20 @@ function IssueReportingPanel({ variant, csrfToken }: { variant: Variant; csrfTok
 export function ExamQuestionStatus({
   instance_question,
   assessment_question,
+  displaySavedAsGradable,
 }: {
   instance_question: InstanceQuestion & {
     allow_grade_left_ms?: number;
     allow_grade_interval?: string;
   };
   assessment_question: Pick<AssessmentQuestion, 'max_auto_points' | 'max_manual_points'>;
+  /**
+   * On exam with mixed real-time grading settings, this flag allows us to
+   * differentiate between questions that are saved and which can be graded by
+   * clicking the "Grade N saved answers" button and those that cannot
+   * (because real-time grading is disabled).
+   */
+  displaySavedAsGradable?: boolean;
```

Comment:
nit: I honestly think this is more clear with just having the option be `realTimeGrading`. I don't have a strong preference though.

Reply:
Its value isn't actually that though:

https://github.com/PrairieLearn/PrairieLearn/blob/2f67b3c4bd8debf01f32e0de96872ba5c7bd24cc/apps/prairielearn/src/pages/studentAssessmentInstance/studentAssessmentInstance.html.ts#L377-L380


IMO `realTimeGrading` would be a misnomer since it's _not_ just that value for a specific question, or even the assessment as a whole, but a specific overall configuration of multiple questions on an assessment.